test/CDB_CartodbfyTableTest.sql

```diff
@@ -281,6 +281,44 @@ INSERT INTO many_colliding_columns VALUES (
 SELECT CDB_CartodbfyTableCheck('many_colliding_columns', 'Many colliding columns #141');
 DROP TABLE many_colliding_columns;
 
+-- table with null cartodb_id
+CREATE TABLE test (
+  cartodb_id integer
+);
+INSERT INTO test VALUES
+  (1), 
+  (2), 
+  (NULL), 
+  (3);
+SELECT CDB_CartodbfyTable('test');
```

Comment:
Use `CDB_CartodbfyTableCheck` instead, that provides extra checks on the resulting table after cartodbfy'ing